library/Cube/IcingaDb/IcingaDbHostStatusCube.php

```diff
@@ -97,4 +90,32 @@ public function getObjectsFilter()
 
         return $this->objectsFilter;
     }
+
+    /**
+     * Fetch the host dimensions
+     *
+     * @param IcingaDbCube $cube
+     *
+     * @return array
+     */
+    public static function fetchHostDimensions(IcingaDbCube $cube): array
```

Comment:
please make this non-static and put it in `IcingaDbCube` (Rename it then to `fetchHostVariableDimensions()`)